packages/plugin-connector/src/build-plugins/query-directory-hmr.js

```diff
@@ -0,0 +1,35 @@
+import fs from 'fs/promises';
+/** @type {import("vite").Plugin} */
+export const queryDirectoryHmr = {
+	name: 'evidence:query-directory-hmr',
+
+	configureServer(server) {
+		server.watcher.add('../../queries/**.sql');
+		server.watcher.addListener(
+			'change',
+			/**
+			 *
+			 * @param {string} filename
+			 * @returns {Promise<void>}
+			 */
+			async (filename) => {
+				if (!(filename.includes('.evidence/template/queries') && filename.endsWith('.sql'))) return;
+				const content = await fs.readFile(filename, 'utf-8');
+				const queryId = filename
```

Comment:
Should probably make a regex for this replace/split/pop menace at some point because I think it pops up in a few places